client/my-sites/domains/map-domain/index.jsx

```diff
@@ -111,6 +113,12 @@ export class MapDomain extends Component {
 		}
 	}
 
+	checkIfDomainIsTransferrable( props ) {
+		if ( props.lastDomainStatus === 'transferrable' ) {
```

Comment:
When you see yourself doing checks against raw strings like this, always try to check if there are already constants you could use. In this case, there are:

https://github.com/Automattic/wp-calypso/blob/fa95e755141615209f2fbb83f64d2775597eace5/client/lib/domains/constants.js#L58

(same for `mappable`)

Reply:
Perhaps I should see if I can change the transfer comparison string as well.